resources/definitions/fdmprinter.def.json

```diff
@@ -1510,11 +1510,11 @@
                 {
                     "label": "Extra Skin Wall Count",
                     "description": "Replaces the outermost part of the top/bottom pattern with a number of concentric lines. Using one or two lines improves roofs that start on infill material.",
-                    "default_value": 1,
+                    "default_value": "0 if top_bottom_pattern == 'concentric' else 1",
```

Comment:
Nope, sorry, you can't do that! default_value is what is used by the engine to get a value. Since it doesn't have a python interpreter, these must be "simple" values (eg not setting functions)

Reply:
noted at changed